app/uk/gov/hmrc/merchandiseinbaggagefrontend/forms/mappings/Formatters.scala

```diff
@@ -52,6 +54,32 @@ trait Formatters {
       def unbind(key: String, value: Boolean) = Map(key -> value.toString)
     }
 
+  private[mappings] def bigDecimalFormatter(
+                                             requiredKey: String,
+                                             nonNumericKey: String,
+                                             args: Seq[String] = Seq.empty): Formatter[BigDecimal] =
+    new Formatter[BigDecimal] {
```

Comment:
Again not for now, but I'm wondering about re-purposing the old CurrencyAmount case class to store a BigDecimal and a String. Then maybe have a Formatter[CurrencyAmount] here instead?